waiter/resources/web/welcome.txt

```diff
@@ -0,0 +1,25 @@
+
+Welcome to Waiter
+=================
+
+  Waiter is a platform for running and managing web services and applications.
+  See `Getting Help' for more information.
```

Comment:
Should we wrap this with `when support-info` so we don't point them to a section that doesn't exist?

Reply:
Done